app/brave_settings_strings.grdp

```diff
@@ -1439,6 +1439,9 @@
   <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_ICON_ON_TOOLBAR_LABEL" desc="The text for settings option">
     Show Leo icon in the sidebar
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_IN_CONTEXT_MENU_LABEL" desc="The text for settings option">
+    Show Leo in the context menu
```

Comment:
I can't find this in the design, but to me it seems like it should be something more explanative, maybe even just a small change like "Show Leo tools in the context menu [on websites]"?

Reply:
fixed in https://github.com/brave/brave-core/pull/21670/commits/d553512aa99fa9b340158c91058f506414c2816d